src/frontend/window/run_mode.rs

```diff
@@ -11,6 +11,7 @@ impl From<Command> for RunMode {
         match value {
             Command::Gui => Self::Gui,
             Command::Tray => Self::Tray,
+            _ => unreachable!("There's  no run mode for {:?} defined.", value),
```

Comment:
why? We only have two Commands. Let the type checker handle this. Otherwise we might forget a future command.

Reply:
eh good point. I was thinking that `_` can be only `Command::PrintConfig` but that's only true for *now* xD